DN1/test/runtests.jl

```diff
@@ -0,0 +1,35 @@
+using DN1
+using Test
+
```

Comment:
Dodajte test izračuna koeficientov. Izračunajte na roke po en primer na 2 in 3 interpolacijskih točkah in preverite, če vaš program vrne iste koeficiente. Primera lahko uporabite tudi za testiranje računanja vrednosti v točkah, ki niso podate kot interpolacijske točke.